.changeset/lazy-roses-drive.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/preview-middleware': patch
+'@sap-ux/ui5-config': patch
+---
+
+Validate ui5 version defined in ui5-proxy-middleware for adp project
```

Comment:
Please fix the description. This is not true. The goal is to validate the UI5 version, regardless from where it is coming.